lib/src/syn/v2/parser/stmt/parts.rs

```diff
@@ -158,7 +158,7 @@ impl Parser<'_> {
 			t!("NONE") => Ok(Permissions::none()),
 			t!("FULL") => Ok(Permissions::full()),
 			t!("FOR") => {
-				let mut permission = Permissions::default();
+				let mut permission = Permissions::none();
```

Comment:
These permissions can stay default. All resources are submissive by default, with the exception of tables which should be restrictive by default

Reply:
Alright! I have change the permissions for field back again.